### ISSUE TITLE
Getting 404 on web_modules/react.js using TypeScript React template 

### ISSUE THREAD

**Issue by mohsen1** [pikapkg/create-snowpack-app#77]
This is the first time I'm using Snowpack. Here is what I did:

My history:

```bash
yarn start
cd new-dir/
npx create-snowpack-app new-dir --template @snowpack/app-template-react-typescript --use-yarn
nvm use 12
cd Desktop/
cd
```

```
GET http://localhost:8080/web_modules/react.js net::ERR_ABORTED 404 (Not Found)
```

I ran all of the commands from VSCode. It prompted me to allow VSCode to control Chrome. I'm on Mac.

Other files are ok. I aw App.js being downloaded fine.

**Comment by stevenpetryk** [comment]
Same, and with the non-TypeScript template too.

**Comment by stevenpetryk** [comment]
Getting the same with Preact. Perhaps it's just an issue with Snowpack... investigating.

**Comment by stevenpetryk** [comment]
Ah yep, I'm only seeing it in 2.0.1. Snowpack 2.0.0 loads all three of them just fine.

Diff between the two: https://github.com/pikapkg/snowpack/compare/v2.0.0...v2.0.1. Lots of potentially breaking things in there as far as I can tell.

**Comment by FredKSchott** [comment]
bah 1 second, looking...

**Comment by FredKSchott** [comment]
Fixed in v1.0.2: https://github.com/pikapkg/snowpack/releases/tag/v2.0.2

Sorry about that! Our dev code is hard to test, but getting something to catch this in the future is priority #1 atm
